operator/pkg/controllers/etcd/persistentvolumeclaim.go

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Do we want to keep PVC related stuff in one file, we already have `persistentvolumeclaimspec.go`. I would just create in one file and call `persistentvolumeclaim.go`

Reply:
good point. Done.